libnmstate/netapplier.py

```diff
@@ -129,11 +129,17 @@ def _apply_ifaces_state(
             autodestroy=commit, timeout=rollback_timeout
         ) as checkpoint:
             with _setup_providers():
-                _add_interfaces(new_interfaces, desired_state)
+                ifaces2add, ifaces_add_configs = _add_interfaces(
```

Comment:
As discussed before, I think that the `_add_interfaces` and `_edit_interfaces` no longer represent well the abstraction they once represented.
But if this shows to bee too challenging, we can fix this later.

Reply:
Yeah. I tried to merged these two functions and trigger a lot issue. We can do that in new PR.